code/src/png.rs

```diff
@@ -168,12 +169,17 @@ mod tests {
 
         chunk_bytes.append(&mut bad_chunk);
 
-        let png = Png::try_from(chunk_bytes.as_ref());
+        let bytes: Vec<u8> = Png::STANDARD_HEADER
+            .iter()
+            .chain(chunk_bytes.iter())
+            .copied()
+            .collect();
```

Comment:
This is giving away some iterator methods.

Reply:
It is copied from [line 126](https://github.com/jrdngr/pngme_book/pull/16/commits/605e0a3068c1548791e7b2e00b7e192a701b6576#diff-447383c680e9270ea4d973dc2a58c855434ab25710878c755759fbd3cec77fe8L126) elsewhere in the file.